src/libs/shared/directives/show-for-logged-in/show-for-logged-in.directive.ts

```diff
@@ -42,8 +43,11 @@ export class ShowForLoggedInDirective implements OnInit {
   }
 
   private changeVisibility(loggedIn: boolean): void {
-    // @TODO: 8) Uzupełnij customową dyrektywę `ShowForLoggedInDirective` która bazując na wartości ze statu o tym
-    //    czy użytkownik jest zalogowany, doda lub usunie element na który została nałożona
+    if (loggedIn && this.showed) {
```

Comment:
generalnie git, brakuje zmiany this.showed, brakuje wykorzystania inputu appShowForLoggedIn